helm-charts/helm3/strimzi-drain-cleaner/Chart.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: v2
-appVersion: 1.1.0
-description: Utility which helps with moving the Apache Kafka pods deployed by Strimzi from Kubernetes nodes which are being drained.
-name: strimzi-drain-cleaner
-icon: https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/main/documentation/logo/strimzi_logo.png
-type: application
-version: 1.1.0
+appVersion: 1.3.0
+description: Utility which helps with moving the Apache Kafka pods deployed by Strimzi
```

Comment:
can we put on one line to minimize changes?

Reply:
that's copy paste from the provided package. IDK why the ordering etc. is changed.